README.md

```diff
@@ -28,60 +28,89 @@ Splunk OpenTelemetry Lambda is a downstream distribution of the [OpenTelemetry L
 
 Currently, following components are supported:
 - [Java wrapper](#java-wrapper) 
+- [Python wrapper](#python-wrapper)
 
 ### Configuration
 
 All wrapper layers are preconfigured to use Splunk as the tracing backend.
 
 1. Context propagation
+
     By default, W3C Trace Context and Baggage (`tracecontext,baggage`) propagators are used. 
     
-    If you want to change this, set the `OTEL_PROPAGATORS` environment variable in your Lambda function code. For more information about available context propagators, see the [Propagator settings](https://github.com/open-telemetry/opentelemetry-java/tree/v1.1.0/sdk-extensions/autoconfigure#customizing-the-opentelemetry-sdk) for the OpenTelemetry Java.
-
-2. Traces export
-    By default, the OpenTelemetry Protocol (`otlp`) exporter is used.
-    If you want to use this exporter, set these environment variables in your Lambda function code:
-    ```
-    OTEL_EXPORTER_OTLP_ENDPOINT: <Endpoint URL>
-    ```
-    Splunk provides also token-authenticated `jaeger-thrift-splunk` exporter for customers that need to use that specific protocol. In order to use it, please set:
-    ```
-    OTEL_TRACES_EXPORTER: jaeger-thrift-splunk
-    OTEL_EXPORTER_JAEGER_ENDPOINT: <Endpoint URL>
-    SPLUNK_ACCESS_TOKEN: orgAccessToken
-    ``` 
-    By default, `OTEL_EXPORTER_JAEGER_ENDPOINT` is set to `http://127.0.0.1:9080/v1/trace` (local SmartAgent endpoint)
+    If you want to change this, set the `OTEL_PROPAGATORS` environment variable in your Lambda function code. 
    
-    Default span flush wait timeout is 1 second. It is controlled with a following environment variable (value in milliseconds):
+2. Traces export
+
+    By default, all wrappers use the Jaeger/Thrift exporter modified by Splunk (`jaeger-thrift-splunk`). The endpoint (`OTEL_EXPORTER_JAEGER_ENDPOINT`) is set to `https://ingest.us0.signalfx.com/v2/trace`, the direct ingest URL for Splunk Observability Cloud. To make it work, you only need to set the Splunk access token:
+     ```
+     SPLUNK_ACCESS_TOKEN: <org_access_token>
+     ``` 
+    If you want to change the endpoint, set this environment variable in your Lambda function code:
     ```
-    OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT: 30000
+    OTEL_EXPORTER_OTLP_ENDPOINT: <endpoint URL>
```

Comment:
Shouldn't it be `OTEL_EXPORTER_JAEGER_ENDPOINT`? Since we're using jaeger-thrift by default
```suggestion
    OTEL_EXPORTER_JAEGER_ENDPOINT: <endpoint URL>
```